install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -294,6 +294,9 @@ type PublicAPIConfig struct {
 	// Name of the kubernetes secret to use for Stripe secrets
 	StripeSecretName string `json:"stripeSecretName"`
 
+	// Name of the kubernetes secret to use for signing JWTs
+	JWTSigningKeySecretName string `json:"jwtSigningKeySecretName"`
```

Comment:
In the `common` config, we use `SecretRef` to refer to secrets. Does it make sense to use the same naming scheme here?

`jwtSigningKeySecretName` is abit opaque, what if we made the secret to be a `oidc-secret` which then contains a couple of keys. We'd mount the secret (and all keys) into a dir, and then point to the dir, the component then knows to look for the `signingKey` file

Reply:
Basically I think the Config should mention it's for OIDC. We can have multiple JWT, or other signing keys and `JWTSigningKeySecretName` doesn't tell me this is to be used for OIDC.

The `PersonalAccessTokenSigningKeySecretName` is not a JWT - this is actually an example of the above. But here, we at least know it relates to PATs

Reply:
> Looking above and below, one can see the -SecretName is a pattern. Sticking to that is consistent.

In the experimental config, it is. But in the `common` config the standard is to use a `SecretRef` - see Database for example

Reply:
> Then we'd need to make them more specific, right?

>  doesn't tell me this is to be used for OIDC.

Alright, I'm adding a prefix to it.

Reply:
Done. It's narrowed down to the use case now. 
```
OIDCClientJWTSigningSecretPath string `json:"oidcClientJWTSigningSecretPath"`
```